theme/templates/base.html

```diff
@@ -196,7 +196,7 @@
                         <h3>{{ siteconf.col1_heading }}</h3>
                     </div>
                     <div class="content">
-                        {{ siteconf.col1_content|richtext_filters|safe }}
+                        {{ siteconf.col1_content|richtext_filters }}
```

Comment:
The default comment filter is basically |urlize|linebreaksbr with autoescaping. It makes text like http://example.com clickable and adds \<br /\>'s , but does do autoescaping of other things. I don't think there's a way to use that default filter and use an anchor text that's different than the URL.